completions/bash/kpod

```diff
@@ -1334,6 +1334,20 @@ _kpod_ps() {
      _complete_ "$options_with_args" "$boolean_options"
 }
 
+_kpod_start() {
+     local options_with_args="
+     --detach-keys
+     "
+
+     local boolean_options="
+     -h
+     --help
+     -a
+     --attach
+     -i
+     --interactive"
+     _complete_ "$options_with_args" "$boolean_options"
+}
 _kpod_stop() {
      local options_with_args="
```

Comment:
You need to add start below in section with all the commands.

Reply:
fixed